src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -229,6 +229,64 @@
     <Compile Include="Interop\SNINativeMethodWrapper.Common.cs" />
     <Compile Include="System\Data\SqlClient\SqlCredential.cs" />
   </ItemGroup>
+  <!-- SqlFileStream -->
+  <!-- Non-Windows -->
+  <ItemGroup Condition=" '$(TargetsWindows)' != 'true' And '$(IsPartialFacadeAssembly)' != 'true' AND '$(OSGroup)' != 'AnyOS'">
+    <Compile Include="System\Data\SqlClient\SqlFileStream.Unsupported.cs" />
+  </ItemGroup>
+  <!-- Windows UAP -->
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' AND '$(IsPartialFacadeAssembly)' != 'true' AND '$(IsUAPAssembly)' == 'true'">
+    <Compile Include="System\Data\SqlClient\SqlFileStream.Unsupported.cs" />
+  </ItemGroup>
+  <!-- Windows Non-UAP dependencies -->
+  <ItemGroup Condition="('$(TargetGroup)' == 'netstandard' OR '$(TargetGroup)' == 'netcoreapp') AND '$(TargetsWindows)' == 'true'">
```

Comment:
Isn't this condition same as line 291? Is there any particular reason, you have duplicated this ItemGroup Condition?